server/src/test/scala/io/delta/sharing/server/DeltaSharingServiceSuite.scala

```diff
@@ -1792,6 +1807,233 @@ class DeltaSharingServiceSuite extends FunSuite with BeforeAndAfterAll {
     verifyPreSignedUrl(actualFiles(2).deltaSingleAction.add.path, 653)
   }
 
+  // Tests that the protocol and metadata from a deletion vector table is correct
+  integrationTest("deletion vector metadata test 1 - /shares/{share}/schemas/{schema}/tables/{table}/metadata") {
+    val response = readNDJson(requestPath("/shares/share8/schemas/default/tables/deletion_vectors_with_dvs_dv_property_on/metadata"), Some("GET"), None, responseFormat = RESPONSE_FORMAT_DELTA, readerFeatures = s";readerFeatures=${DeletionVectorsTableFeature.name}")
+    val lines = response.split("\n")
+    val protocol = lines(0)
+    val metadata = lines(1)
+
+    verifyDeltaResponseProtocolAndMetadata(
+      actualProtocolStr = protocol,
+      actualMetadataStr = metadata,
+      minReaderVersionProtocol = 3,
+      minWriterVersionProtocol = 7,
+      readerFeaturesProtocol = Option(Set("deletionVectors")),
+      writerFeaturesProtocol = Option(Set("deletionVectors")),
+      enableDeletionVectors = "true",
+      checkpointInterval = "10000",
+      idMetadata = "7f10249f-e3ff-4fe4-967a-05637934a7e9",
+      descriptionMetadata = Option("Deletion vectors enabled table. The latest version has DVs turned on and DVs present in the table."),
+      providerMetadata = RESPONSE_FORMAT_PARQUET,
+      schemaStringMetadata = """{"type":"struct","fields":[{"name":"id","type":"long","nullable":true,"metadata":{}},{"name":"value","type":"string","nullable":true,"metadata":{}},{"name":"timestamp","type":"timestamp","nullable":true,"metadata":{}},{"name":"rand","type":"double","nullable":true,"metadata":{}}]}""",
+      createdTimeMetadata = Option(1685559511124L)
+    )
+  }
+
+  // Tests that the protocol and metadata from a deletion vector table is correct
+  // for responseFormat containing Delta and Parquet
+  integrationTest("deletion vector metadata test 2 - /shares/{share}/schemas/{schema}/tables/{table}/metadata") {
```

Comment:
Can we combine two test cases into one like below? Since except for `responseFormat` the rest are the same.
```suggestion
  integrationTest("deletion vector metadata test 2 - /shares/{share}/schemas/{schema}/tables/{table}/metadata") {
    Seq(RESPONSE_FORMAT_DELTA, parquetAndDeltaResponse).foreach {
     ...
    }
```

Reply:
fixed!